checkQC/default_config/config.yaml

```diff
@@ -20,8 +20,8 @@
 
 default_handlers:
     - name: UndeterminedPercentageHandler
-      warning: 10
-      error: 30
+      warning: unknown
+      error: 9
```

Comment:
Should we set this to 10 since the "default" amount of PhiX for most runs is 1%?

Reply:
Good point. Will do.